### PR TITLE
1.0 Cleans

### PR DESCRIPTION
**Pull request by TomDonoghue**
This is just a brief check & clean through the repo of the current 1.0 codebase. 

@ryanhammonds - I've tagged you for review, but it's really just for a super quick check I didn't do anything silly - everything should be pretty straight-forward.

### CODE REVIEW

**Comment by TomDonoghue** [comment]
@ryanhammonds - I fixed the merge conflicts here after the other PRs. 
Should be good to go - feel free to merge it in if this looks good to you!

File: bycycle/group/features.py [hunk 1]
```diff
@@ -40,9 +40,9 @@ def compute_features_2d(sigs, fs, f_range, compute_features_kwargs=None, axis=0,
     return_samples : bool, optional, default: True
         Whether to return a dataframe of cyclepoint sample indices.
     n_jobs : int, optional, default: -1
-        The number of jobs, one per cpu, to compute features in parallel.
-    progress: {None, 'tqdm', 'tqdm.notebook'}
-        Specify whether to display a progress bar. Use 'tqdm' if installed.
+        The number of jobs, one per CPU, to compute features in parallel.
```
Comment (ryanhammonds) [inline bycycle/group/features.py:43]: Maybe "one per CPU" should be removed. Some hardware is hyperthreaded to allow for 2 jobs per CPU.

**Review by ryanhammonds — COMMENTED** [review]
Thanks for this! Everything looks good here. I'm going to read through all 1.0 changes today and push updates if I find any typos/etc here, then merge.

File: bycycle/group/features.py (hunk 1 above)
Comment (TomDonoghue) [reply bycycle/group/features.py:43]: Yeh, that's a good point! I think it makes more sense to refer to number of jobs, without stating / assuming anything about CPUs.

**Pull request merged**